x-pack/legacy/plugins/canvas/public/components/workpad_header/workpad_header.tsx

```diff
@@ -194,11 +194,14 @@ export class WorkpadHeader extends React.PureComponent<Props, State> {
                 <EuiFlexItem grow={false}>
                   <AssetManager />
                 </EuiFlexItem>
+                {/*
+                Disabling embeddables until we figure out filters
```

Comment:
Same.  And tag with `TODO`.  Consider `TODO: elastic/kibana#12345 Disabling pending filters work`